packages/amplify_core/lib/src/category/amplify_push_notifications_category.dart

```diff
@@ -9,6 +9,10 @@ class PushNotificationsCategory
   @nonVirtual
   Category get category => Category.pushNotifications;
 
+  @override
+  @nonVirtual
+  Set<Category> get categoryDependencies => const {Category.analytics};
```

Comment:
Does push notifications need analytics to function?  I thought it was setup to work independently by importing in analytics code but not relying on the analytics category itself.